Source/Core/Core/BootManager.cpp

```diff
@@ -98,7 +98,6 @@ bool BootCore(const std::string& _rFilename)
 		return false;
 
 	// Load game specific settings
-	if (StartUp.GetUniqueID().size() == 6)
 	{
```

Comment:
Is it okay to keep the braces like this? I thought it would be appropriate to get rid of the IniFile object once it won't be used anymore.